xpdan/data_reduction.py

```diff
@@ -149,7 +151,7 @@ def _file_name(self, event, event_timestamp, ind):
 # init
 xpd_data_proc = DataReduction()
 ai = AzimuthalIntegrator()
-
+geo = Geometry()
```

Comment:
👍  I first did that then I reverted my commit in order to keep your code intact. I will do it then.